app/sentinel/internal/safe/post.go

```diff
@@ -100,14 +80,14 @@ func Post(workloadId, secret, namespace, backingStore string, useKubernetes bool
 		}
 	}
 
-	f := data.None
+	f := data.Json
```

Comment:
Switching the default to `Json` due to a discussion elsewhere.

Also removing `data.None` as a type, for extra simplicity.

Did a decision-tree analysis, and at any branch `data.None` was equivalent to either `data.Json` or `data.Yaml`: So there is no need to hav eit.